src/idv_service/session/retrieve/get.session.result.js

```diff
@@ -94,6 +95,19 @@ class GetSessionResult {
       /** @private */
       this.identityProfile = new IdentityProfileResponse(response.identity_profile);
     }
+
+    if (response.advanced_identity_profile) {
+      Validation.isPlainObject(response.advanced_identity_profile, 'advanced_identity_profile');
+      console.log('advanced_identity_profile:');
+      /** @private */
+      // eslint-disable-next-line max-len
+      this.advancedIdentityProfile = new AdvancedIdentityProfileResponse(response.advanced_identity_profile);
+    }
+
+    if (response.advanced_identity_profile_preview) {
+      console.log('advanced_identity_profile_preview:');
+      console.log(JSON.stringify(response.advanced_identity_profile_preview, null, 2));
+    }
```

Comment:
Probably this whole section needs to be removed. I haven't seen advanced_identity_profile_preview in the response or maybe one to confirm with Pawel.